packages/x-data-grid-premium/src/hooks/features/aggregation/wrapColumnWithAggregation.tsx

```diff
@@ -47,7 +56,8 @@ const getAggregationValueWrappedValueGetter: ColumnPropertyWrapper<'valueGetter'
   getCellAggregationResult,
 }) => {
   const wrappedValueGetter: GridBaseColDef['valueGetter'] = (value, row, column, apiRef) => {
-    const rowId = apiRef.current.getRowId?.(row);
+    const { getRowId: getRowIdProp } = gridPropsStateSelector(apiRef.current.state);
+    const rowId = getRowId(row, getRowIdProp);
```

Comment:
I think this could be simplified to a single `gridRowIdSelector(apiRef, row)`, and the `getRowId` logic would be inside the selector instead of outside.

Reply:
Great suggestion. Thanks.